### PR TITLE
NullPointerException when creating a watch with Jira action (#41922)

### PR DESCRIPTION
**Pull request by ojasgulati**
NullPointerException when secured_url does not use proper scheme in jira action. This commit will handle Expection and display proper message.

### CODE REVIEW

**Comment by ojasgulati** [comment]
@ielatif i made the changes please have a look.

**Comment by ielatif** [comment]
Relates to #41922

**Review by ielatif — CHANGES_REQUESTED** [review]

File: x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/common/http/Scheme.java
Comment (ojasgulati) [reply x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/common/http/Scheme.java]: Made correction. please have a look 😃

**Review by ielatif — APPROVED** [review]

**Comment by jbaiera** [comment]
Thanks for opening this PR @ojasgulati. 

@elasticmachine Test this please

**Comment by jbaiera** [comment]
@ojasgulati CI is failing on what looks like some unrelated issues. Could you merge upstream master into your branch just to make sure CI is all green before merging? Alternatively, I can do the merge and CI dance if you [enable maintainer contributions](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) on this PR

**Comment by ojasgulati** [comment]
@jbaiera I am unable to merge. Newbie to git 😅. It would be very helpful if you merge the upstream master. I have already enabled edits from maintainers. Also if possible, could you please share a link to a good tutorial on how to merge upstream so I can deal with this problem in future. 
Thanks

**Comment by ojasgulati** [comment]
@jbaiera I have merged the upstream master please have a look.

**Comment by jbaiera** [comment]
Thanks @ojasgulati, appreciate it!

@elasticmachine Test this please

**Review by jbaiera — APPROVED** [review]
LGTM

**Pull request merged**
